.vscode/launch.json

```
@@ -110,7 +110,7 @@
             "program": "${env:OutRoot}Winx64/Product/Units-Gtest/UnitsTest.exe",
             "args": [
                 "--gtest_break_on_failure",
-                "--gtest_filter=CompositeValueSpecJsonTest.TestDefaultAndEmptySpacerRoundTrips"
+                "--gtest_filter=FormattingTestFixture.DoubleToRatio"
```

```suggestion
                "--gtest_filter=CompositeValueSpecJsonTest.TestDefaultAndEmptySpacerRoundTrips"
```
remove change here, keep local